### PR TITLE
Udit/readme external

### PR DESCRIPTION
**Pull request by uditmehta27**
PR updates the README with instructions to start using the `workspaceID` as the `externalID`. It also contains steps to migrate for existing customers.

### CODE REVIEW

**Review by f2prateek — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ terraform help
 mkdir segment-datalakes-tf
 ```
 * Create `main.tf` file
-    * Update the `segment_sources` variable in the `locals` to the sources you want to sync
+    * Update the `external_ids` variable in the `locals` to the workspace ID. This will allow all sources in the workspace to be synced to the Data Lake without any extra setup.
```
Comment (ynguyensegment) [inline README.md:51]: > Update the `external_ids` variable in the `locals` to the workspace ID
Is a noun missing after `locals`?

**Review by maggieyu-segment — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -48,7 +48,8 @@ terraform help
...
+      * **Note - Existing users** may be using the `sourceID` here instead of the `workspaceID`, which was the previous configuration. Setting this value as the `sourceID` is still supported for existing users for backwards compatibility. Follow instruction [here](#Using-workspaceID-as-externalID) to migrate to `workspaceID`. This will ensure you do not need to update this value for each source you want to add.
```
Comment (ynguyensegment) [inline README.md:52]: the `#Using-workspaceID-as-externalID` link doesn't seem to be working in the preview view

Comment (uditmehta27) [reply README.md]: Yeah ill fix this in a follow-up update once I merge and get the link to the section.

File: README.md (hunk 1 above)
Comment (uditmehta27) [reply README.md]: done!

**Review by ynguyensegment — APPROVED** [review]

**Pull request merged**